src/serial/serialcommands.cpp

```diff
@@ -147,6 +148,10 @@ namespace SerialCommands {
     }
 
     void cmdGet(CmdParser * parser) {
+#if USE_REMOTE_COMMAND && ALLOW_REMOTE_WIFI_PROV
+		if (cmdFromRemote && !networkConnection.isConnected()) return;
+#endif
```

Comment:
This is used multiple times throughout this file, but it's not applied to every command? Why not just put it at the parser before the command functions are called? Also why is it on get and such? Isn't WiFi prov just for setting the WiFi?

Reply:
The idea here is to only allow `SET` and block other commands in the provisioning phase, before connected to a slime server. The repetition is from that the command is only known to be a `SET` after parsing, which is processed inside cmdParser.